packages/frontend/src/renderer/sagas/notifications/notifications.test.ts

```diff
@@ -82,7 +81,7 @@ beforeAll(async () => {
       createdAt: 1000000,
       channelAddress: publicChannel2.channel.address,
       signature: 'signature',
-      pubKey: 'pubKey22'
+      pubKey: Object.keys(users.selectors.certificatesMapping(store.store.getState())).find((pubKey) => pubKey !== userPubKey)
```

Comment:
Please define this value higher in the tests, assigning things to separate variables (improve readability)